.github/workflows/fuzzer-tester-prepare/action.yml

```diff
@@ -22,10 +22,6 @@ runs:
     - name: Add wasm target
       shell: bash
       run: rustup target add wasm32-unknown-unknown
-    - name: Remove obsolete llvm (Linux)
```

Comment:
@addisoncrump 
because of this man

Reply:
if you don't remove llvm then the obsolete llvm will still be used

Reply:
Got it, thanks.

Reply:
yes because of llvm 14 doesn't work.
but with llvm 14 libafl_libfuzzer won't even compile (not just link) to begin with

Reply:
No worries! I just happened to need this test for #2061 so I fired it through :sweat_smile: